src/main/java/org/folio/bulkops/service/ErrorService.java

```diff
@@ -116,9 +113,11 @@ public void saveErrorsFromDataImport(UUID bulkOperationId, UUID dataImportJobId)
         String identifier = EMPTY;
         try {
           if (identifierType == IdentifierType.ID) {
-            identifier = srsClient.getSrsRecordById(errorEntry.getSourceRecordId()).getExternalIdsHolder().getInstanceId();
+            var idList = errorEntry.getRelatedInstanceInfo().getIdList();
+            identifier = idList.isEmpty() ? null : idList.get(0);
           } else if (identifierType == IdentifierType.INSTANCE_HRID) {
-            identifier = srsClient.getSrsRecordById(errorEntry.getSourceRecordId()).getExternalIdsHolder().getInstanceHrid();
+            var hridList = errorEntry.getRelatedInstanceInfo().getHridList();
+            identifier = hridList.isEmpty() ? null : hridList.get(0);
           }
         } catch (Exception e) {
           log.error("Problem with retrieving SRS record {}", errorEntry.getSourceRecordId(), e);
```

Comment:
Is this error message still valid taking into account call to SRS is removed?

Reply:
Yes, you're right. Removed, thank you.